home/palettes.asm

```diff
@@ -1,15 +1,24 @@
 ; Functions dealing with palettes.
 
+UpdateCGBPalsLYTimed:
+; checks if there is time to run UpdateCGBPals.
+	ldh a, [rLY]
+	cp 150
+	ret nc
+
+; fallthrough
```

Comment:
Keeping the comment near the thing that falls through, where a `ret`/`jp` would otherwise go:
```suggestion
	; fallthrough

```